packages/design-system/accordion/index.tsx

```diff
@@ -57,7 +57,7 @@ const Trigger = ({ tw = "", ...props }: ViewProps) => {
   return (
     <RNAccordion.Trigger>
       <View
-        tw={"w-full flex-row items-center justify-between px-4 py-4 " + tw}
+        tw={"w-full flex-row items-center justify-between px-4 py-5 " + tw}
```

Comment:
Nit: prefer using an array here instead of string concatenation

Reply:
yeah, also like array style!